README.md

```diff
@@ -396,7 +396,7 @@ to a verilator simulation model to be simulated on a PC.
 ```
 mkdir sw/build
 pushd sw/build
-cmake ../
+cmake .. -DCMAKE_TOOLCHAIN_FILE=../gcc_toolchain.cmake -DCMAKE_BUILD_TYPE=Debug
```

Comment:
Is there any way to keep the command as `cmake ..` and have the `gcc_toolchain.cmake` be included by default?

Reply:
Sure thing, I set `gcc_toolchain.cmake` as the default in the top-level `CMakeLists.txt`. It can still be overridden if `-DCMAKE_TOOLCHAIN_FILE` is specified.

Reply:
Thanks that looks a lot cleaner.